package.json

```diff
@@ -24,15 +24,32 @@
         "url": "https://github.com/DEGJS/breakpoints.git"
     },
     "bugs": "https://github.com/DEGJS/breakpoints/issues",
-    "jspm": {
-        "registry": "jspm",
-        "format": "es6",
-        "main": "breakpoints.js",
-        "directories": {
-            "lib": "src"
-        },
-        "dependencies": {
-            "DEGJS/eventAggregator": "github:DEGJS/eventAggregator@^1.0.2"
-        }
+    "peerDependencies": {
+        "@degjs/event-aggregator": "^2.0.0"
```

Comment:
Breakpoints uses event aggregator directly, so this should be a dependency rather than a peer dependency